### ISSUE TITLE
Correctly draw the white rectangle in the world map window

### ISSUE THREAD

**Issue by kazgaroth011** [SWY1985/CivOne#404]
When at the bottom of the screen, the rectangle should be positioned slightly more down (it's drawn correctly for the northern part I think).

![clipboard04](https://user-images.githubusercontent.com/21365802/29229996-533911fa-7ee1-11e7-8470-2cd62e230936.png)

**Comment by SWY1985** [comment]
The square is incorrectly sized in the actual game as well... I will consider this a bug in the original game and fix it.
While I'm fixing this, I will also adjust the white square to reflect the screen size, with the expanded window feature that I'm working on.

**Comment by kazgaroth011** [comment]
Yes I've also noticed it's broken in the original. Especially the positioning of the rectangle when you're moving with the mouse at the bottom of the map.

**Comment by SWY1985** [comment]
Moved back to alpha.2.
